src/code/InstallHelper.cs

```diff
@@ -97,7 +99,8 @@ public InstallHelper(PSCmdlet cmdletPassedIn)
                 noClobber.ToString(),
                 asNupkg.ToString(),
                 includeXml.ToString(),
-                savePkg.ToString()));
+                savePkg.ToString(),
+                tmpPath != null ? tmpPath.ToString() : string.Empty));

```

Comment:
`ToString()` is not needed since `tmpPath` is a string type.
```suggestion
                tmpPath ?? string.Empty));

```